keps/sig-api-machinery/3157-watch-list/README.md

```diff
@@ -797,23 +802,29 @@ Pick one of these and delete the rest.
 
 - [x] Feature gate (also fill in values in `kep.yaml`)
   - Feature gate name: WatchList
-  - Components depending on the feature gate: the kube-apiserver
+  - Components depending on the feature gate: 
+    - kube-apiserver
+    - kube-controller-manager
 - [ ] Other
   - Describe the mechanism:
   - Will enabling / disabling the feature require downtime of the control
-    plane?
+    plane? 
+Yes.
+After graduation to beta the feature will be automatically enabled for kube-apiserver and kube-controller-manager.
+Disabling requires passing `WatchList=false` to `feature-gates` command line flag for both components.
   - Will enabling / disabling the feature require downtime or reprovisioning
-    of a node?
+    of a node? No.
 
 ###### Does enabling the feature change any default behavior?
-No.
+No. User have to explicitly set `ENABLE_CLIENT_GO_WATCH_LIST_ALPHA` environment variable.
 <!--
 Any change of default behavior may be surprising to users or break existing
 automations, so be extremely careful here.
 -->
 
 ###### Can the feature be disabled once it has been enabled (i.e. can we roll back the enablement)?
-Yes, in that scenario the kube-apiserver will reject WATCH requests with the new query parameter forcing informers to fall back to the previous mode.
+Yes, by passing `WatchList=false` to `feature-gates` command line flag to the components for which this feature was enabled. 
```

Comment:
See my comment above.

[Also, you don't need to mention how FG is disabled - just say that by disabling WatchList FG]